packages/emotion/src/withStyle.tsx

```diff
@@ -31,6 +31,20 @@ import { useTextDirectionContext } from '@instructure/ui-i18n'
 import { bidirectionalPolyfill } from './styleUtils/bidirectionalPolyfill'
 import { getComponentThemeOverride } from './getComponentThemeOverride'
 import { useTheme } from './useTheme'
+
+export interface MakeStyleConfig {
+  // The previous prop object that will be compared to the current props
+  prevProps: Record<string, unknown>
+  // An array of property names used in styles.js. The style object will
+  // recalculate if any of these properties change In cannot be used to compare
+  // nested properties.
```

Comment:
Imho we should be consistent with the `interface` and `type` usage.
By default I think we should stick with the `type` and if we happen to be in a situation where we would need to add a field to a type, then an interface could be used.
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Reply:
OK